### PR TITLE
1485 task8 responsive design enhancement

### PR DESCRIPTION
**Pull request by Sanaiqbal-dev**
Task 8 is complete.
User Story : https://dev.azure.com/CodeNinjaConsulting/Talent%20Pool/_workitems/edit/1426

### CODE REVIEW

File: styles.css
Comment (huzaifah0x00) [inline styles.css:1]: Can you please break this file down into multiple css files. having only one css file makes it very hard to maintain and read.

it seems you have different pages in the project ( data view, signup, portfolio, etc ), you should have separate file for 
different pages and include that css with <link> in each html file.

if you have some styles that need to be used by multiple files, you can put those in a common css file.
but css that is only used by a specific page should only be included in that html page.

**Review by huzaifah0x00 — CHANGES_REQUESTED** [review]

File: styles.css
Comment (huzaifah0x00) [reply styles.css:1]: I will review the rest of the PR once this is done.

**Review by huzaifah0x00 — APPROVED** [review]
@Sanaiqbal-dev I've approved the pull request, the responsivess part looks good :thumbsup: 
One thing though, I noticed a bug that once you edit and add new experiences, the new experiences don't show up in the homepage, only in the edit page. ( please let me know if I'm wrong ). This is something you should fix ( not necessary as part of this PR, feel free to merge this one and create a new PR for that later ).

**Comment by Sanaiqbal-dev** [comment]
> @Sanaiqbal-dev I've approved the pull request, the responsivess part looks good 👍 One thing though, I noticed a bug that once you edit and add new experiences, the new experiences don't show up in the homepage, only in the edit page. ( please let me know if I'm wrong ). This is something you should fix ( not necessary as part of this PR, feel free to merge this one and create a new PR for that later ).

**Comment by Sanaiqbal-dev** [comment]
> > @Sanaiqbal-dev I've approved the pull request, the responsivess part looks good 👍 One thing though, I noticed a bug that once you edit and add new experiences, the new experiences don't show up in the homepage, only in the edit page. ( please let me know if I'm wrong ). This is something you should fix ( not necessary as part of this PR, feel free to merge this one and create a new PR for that later ).

**Comment by Sanaiqbal-dev** [comment]
> @Sanaiqbal-dev I've approved the pull request, the responsivess part looks good 👍 One thing though, I noticed a bug that once you edit and add new experiences, the new experiences don't show up in the homepage, only in the edit page. ( please let me know if I'm wrong ). This is something you should fix ( not necessary as part of this PR, feel free to merge this one and create a new PR for that later ).

Thanks for approving the PR, and pointing out a potential bug. I have noticed it while working on my current task and the bug is related to it. I'll make sure that it is fixed in next PR.

**Pull request merged**